src/lib/date.ts

```diff
@@ -14,7 +14,60 @@ const formattedOpeningHoursDate = date => {
   }
 }
 
+export function formattedHours(startAt, endAt) {
+  // If within same am/pm:   9 – 11:30pm
+  // If within across both am/pm:   9am – 12:30pm
+  const startMoment = moment(startAt)
+  const endMoment = moment(endAt)
+  let startHour
+  let endHour
+  if (
+    (startMoment.hours() <= 12 && endMoment.hours() <= 12) ||
+    (startMoment.hours() >= 12 && endMoment.hours() >= 12)
+  ) {
+    // within same am/pm
+    if (startMoment.minutes() === 0) {
+      startHour = startMoment.format("h")
+    } else {
+      startHour = startMoment.format("h:mm")
+    }
+  } else {
+    if (startMoment.minutes() === 0) {
+      startHour = startMoment.format("ha")
+    } else {
+      startHour = startMoment.format("h:mma")
+    }
+  }
+
+  if (endMoment.minutes() === 0) {
+    endHour = endMoment.format("ha")
+  } else {
+    endHour = endMoment.format("h:mma")
+  }
+
+  return `${startHour} – ${endHour}`
+}
+
+export function datesAreSameDay(startAt, endAt) {
+  // Returns true if dates are on same day
+  const startMoment = moment(startAt)
+  const endMoment = moment(endAt)
+  if (
+    startMoment.dayOfYear() === endMoment.dayOfYear() &&
+    startMoment.year() === endMoment.year()
+  ) {
+    // Duration is the same day
+    return true
+  } else {
+    // Duration is a range
+    return false
+  }
+}
+
 export function formattedOpeningHours(startAt, endAt) {
+  // Opens Mar 29 at 4pm
+  // Closes Apr 3 at 12:30pm
+  // Closed
```

Comment:
Could this comment be expanded slightly, perhaps in a docblock form? Something that makes it a bit clearer rather than floating strings.